dbms/src/Flash/tests/gtest_executor.cpp

```diff
@@ -59,6 +59,16 @@ class ExecutorTestRunner : public DB::tests::ExecutorTest
                              {toVec<String>("s", {"banana", "banana"}),
                               toVec<String>("join_c", {"apple", "banana"})});
     }
+
+    void executeExecutor(const std::shared_ptr<tipb::DAGRequest> & request, const ColumnsWithTypeAndName & expect_columns, size_t concurrency = 1)
+    {
+        std::vector<String> enable_planners{"true", "false"};
```

Comment:
should we move `std::vector<String> enable_planners{"true", "false"}` to `executeStreams`?
@ywqzzy